ui/src/app/views/initial-error/component.ts

```diff
@@ -1,5 +1,5 @@
 import {Location} from '@angular/common';
-import {Component, Injectable} from '@angular/core';
+import {Component} from '@angular/core';
```

Comment:
While you're cleaning up TS things, per my comment on your PR please make `loc` public so that the template can properly see it (not sure why the TS compiler isn't complaining about that).

Reply:
Done!